src/io/flutter/run/daemon/DeviceDaemon.java

```diff
@@ -87,7 +87,14 @@ boolean isRunning() {
    * <p>This is calculated based on add and remove events seen since the process started.
    */
   ImmutableList<FlutterDevice> getDevices() {
-    return devices.get();
+    ImmutableList<FlutterDevice> attachedDevices = devices.get();
+    if (attachedDevices.size() > 1) {
```

Comment:
We may want `>= 1`, in order for consistency in the UI when devices are being connected and disconnected.